papaparse.js

```diff
@@ -415,6 +415,50 @@
 		}
 	}
 
+	function DynamicTypingConfig(config)
+	{
+		this._columnConfig = {};
+		this._isAllDynamic = false;
+		this._isBlackListOnly = true;
+
+		if (config === true)
+		{
+			this._isAllDynamic = true;
+			return;
+		}
+		else if (!config)
+		{
+			return;
+		}
+
+		if (config.__only instanceof Array)
+		{
+			this._isBlackListOnly = false;
+			config.__only.forEach(function(field)
+			{
+				this._columnConfig[field] = true;
+			}.bind(this));
+		}
+		else
+		{
+			this._isBlackListOnly = !!config.__except;
+			this._columnConfig = config;
```

Comment:
Needed to be backwards compatible with the old way of configuring. Again, see  [here](https://github.com/vyrak/PapaParse/blob/98dc2405a72c4598a599f96c946c61758a070e55/tests/test-cases.js#L664) &  [here](https://github.com/vyrak/PapaParse/blob/98dc2405a72c4598a599f96c946c61758a070e55/tests/test-cases.js#L682) for use cases.